cpp/ql/src/semmle/code/cpp/commons/Synchronization.qll

```diff
@@ -171,7 +169,7 @@ class DefaultMutexType extends MutexType {
   }
 }
 
-/** Get the mutex argument of a call to lock or unlock. */
+/** Holds if `arg` is the mutex argument of a call to lock or unlock. */
```

Comment:
What about `argType`? All the other predicates seem to not have such a parameter, so it might be a good idea to provide a description of what this represents.

Reply:
I've now added a bit about `argType`.